pywbemtools/pywbemcli/pywbemcli.py

```diff
@@ -786,7 +795,7 @@ def create_server_instance(connection_name):
             format(_python_nm[0], _python_nm[1]),
             DeprecationWarning)
 
-    # Invoke command if one exists.
+    # Invoke command if one exists or start the repl mode
```

Comment:
This is an unrelated change.

Reply:
See comment above.  Not a change but correction of the comment.

Reply:
That is a change. See my response above.